### ISSUE TITLE
Enforce unique Study names in a same Investigation

### ISSUE THREAD

**Issue by rabuono** [ELIXIR-Belgium/seek#31]
To avoid confusion when navigating and searching, but also conflict in an ISAjson export, it would be good if DataHub would enforce unique Study names linked to a same Investigation.
Otherwise multiple different studies with the same `title` and `filename` (in the ISAjson file) might be referred to in a same Investigation.

**Comment by vdk88** [comment]
We might need to enable a study name uniqueness validator when **DataHub** features are switched on in the settings.

**Comment by rabuono** [comment]
Agreed. I assume that the rule of having unique study names inside one investigation (and not necessarily across investigations) will be enough.
